docs/e2e-testing.md

```diff
@@ -54,39 +54,39 @@ In the following section, you'll learn how to build your own E2E test with Cypre
    │   ├── plugins
    ```
 
-Now you're ready to start developing your own E2E tests! Additionally, you can install the VTEX helper functions hub by running (command - WIP) to make writing E2E tests easier.
+Now you're ready to start developing your own E2E tests! Additionally, you can install the [VTEX helper functions hub](https://github.com/vtex/test-toolshttps://github.com/vtex/test-tools) to make writing E2E tests easier.
 
 ### Step 2: Writing a sample test
 
 In the following, we'll create a simple test that verifies how a specific product search behaves. Use it as an example to create your own E2E test with Cypress.
 
 1. Open up your SFJ project in the code editor of your choosing.
 2. Create a new file named `index.js` inside `cypress/e2e/{test-name}` (e.g., `cypress/e2e/search`).
-3. Using JavaScript, write your E2E test. Take the code below as an example:
+3. Make sure to add the `data-testid` attribute to the elements that will need some interaction. In our case are `searchBarInput`, `searchBarItem`, and `productTitle`.
+4. Using JavaScript, write your E2E test. Take the code below as an example:
```

Comment:
Why we don't recommend using TS?

Reply:
I think it doesn't matter if it's JS or TS. It's like "use JS in anyhow". [There's support to TS](https://docs.cypress.io/guides/tooling/typescript-support) anyway.